src/Sets/SimpleSparsePolynomialZonotope.jl

```diff
@@ -152,3 +152,45 @@ The set resulting from applying the linear map `M` to `P`.
 function linear_map(M::AbstractMatrix, P::SSPZ)
     return SimpleSparsePolynomialZonotope(M * center(P), M * genmat(P), expmat(P))
 end
+
+"""
+    quadratic_map(Q::Vector{MT}, S::SimpleSparsePolynomialZonotope) where {N, MT<:AbstractMatrix{N}}
+
+Return an overapproximation of the quadratic map of the given zonotope.
+
+### Input
+
+- `S` -- simple sparse polynomial zonotope
+- `Q` -- array of square matrices
+
+### Output
+
+An overapproximation of the quadratic map of the given zonotope.
```

Comment:
Please fill out with whatever the source is. If the formula is from the slides, we should check if one of the options below comes close to it.

```suggestion
An overapproximation of the quadratic map of the given zonotope.

### Algorithm

This method implements Proposition 12 in [1].
See also Proposition 3.1.30 and Proposition 3.1.31 in [2].

[1] Kochdumper, Althoff. *Sparse polynomial zonotopes - a novel set
representation for reachability analysis*. 2021
[2] Kochdumper. *Extensions of polynomial zonotopes and their application to
verification of cyber-physical systems*. 2021.
```